python/tvm/build_module.py

```diff
@@ -290,12 +293,15 @@ def get_binds(args, binds=None):
     arg_list = []
     for x in args:
         if isinstance(x, tensor.Tensor):
+            any_dim = any(isinstance(i, expr.Var) for i in x.shape)
+            buffer_type = "auto_broadcast" if any_dim and not compact else ""
```

Comment:
Just curious, what does it mean when any_dim and compact are both true?

Reply:
`any_dim` indicates if it is symbolic. Here, we don't really distinguish relay Any and tvm Var. `compact` means that a buffer has already declared as compact type before (i.e. with attribute attr::buffer_bind_scope) and bound to the variable `x` using `AttrStmt`. If so, we shouldn't not declare a strided buffer and bind it to `x`.